spec.html

```diff
@@ -5638,6 +5638,22 @@ <h1>IsStringPrefix ( _p_, _q_ )</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-isstringvalidunicode" aoid="IsStringValidUnicode">
+      <h1>Static Semantics: IsStringValidUnicode ( _string_ )</h1>
```

Comment:
The Unicode concept is "well-formed", and note that it applies to sequences of code _units_ purporting to be in a particular encoding form: http://www.unicode.org/glossary/#well_formed_code_unit_sequence